packages/webdriverio/src/scripts/shadowDom.ts

```diff
@@ -0,0 +1,27 @@
+export async function findIframeInShadowDOM(
+    browser: WebdriverIO.Browser,
+    urlFragment: string
+): Promise<WebdriverIO.Element | null> {
+    return await browser.execute(function (fragment: string) {
```

Comment:
scripts should only contain what is being executed in browser, e.g.

```ts
export default function findIframeInShadowDOM (fragment: string) {
  ...
}
```

and then used like this:

```ts
import { findIframeInShadowDOM } from '../../scripts/shadowDom.js'

await browser.execute(findIframeInShadowDOM, urlFragment)
```